src/views/transactions/components/fee/fee.view.jsx

```diff
@@ -26,7 +25,6 @@ function Fee({
   preferredCurrency,
   fiatExchangeRates,
   showInFiat,
-  tokensPriceTask,
```

Comment:
Since `tokensPriceTask` is not required, we should remove it from `src/views/transactions/components/transaction-form/transaction-form.view.jsx` when we instantiate the `Fee` component.

Reply:
Right! I've removed it from all the flows also